Dockerfile

```diff
@@ -2,17 +2,17 @@ FROM python:3.10 as base
 
 WORKDIR /app
 
-COPY sched_slack_bot sched_slack_bot
-COPY bin/app.py app.py
-COPY poetry.lock .
-COPY pyproject.toml .
-COPY setup.py .
-
 RUN apt-get -y install curl
 RUN curl -sSL https://install.python-poetry.org/install-poetry.py | python -
 ENV PATH=/root/.local/bin:$PATH
 RUN poetry config virtualenvs.create false
 
+COPY poetry.lock .
+COPY pyproject.toml .
+COPY setup.py .
 RUN poetry install --no-dev --no-interaction --no-ansi
 
-ENTRYPOINT python app.py
+COPY sched_slack_bot sched_slack_bot
+COPY bin/app.py app.py
```

Comment:
Drive-by: Re-order Dockerfile to not install everything from scratch if source file changes.